lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -115,9 +114,9 @@ sub livelog {
     # logfile and send them to the client, plus a utility function to
     # close the connection if anything goes wrong.
     my $id;
-    my $close = sub {
+    local *doclose = sub {
```

Comment:
This looks a bit awkward to me, what makes the glob a better choice than a lexical scalar here?